build/ci/.goreleaser.yml

```diff
@@ -27,16 +27,14 @@ changelog:
 dockers:
   -
     image_templates:
-      - quay.io/spaces/kube-httpcache:latest
-      - quay.io/spaces/kube-httpcache:v{{ .Major }}
-      - quay.io/spaces/kube-httpcache:v{{ .Major }}.{{ .Minor }}
-      - quay.io/spaces/kube-httpcache:{{ .Tag }}
-      - quay.io/spaces/kube-httpcache:stable
+      - quay.io/mittwald/kube-httpcache:latest
+      - quay.io/mittwald/kube-httpcache:v{{ .Major }}
+      - quay.io/mittwald/kube-httpcache:v{{ .Major }}.{{ .Minor }}
+      - quay.io/mittwald/kube-httpcache:{{ .Tag }}
+      - quay.io/mittwald/kube-httpcache:stable
```

Comment:
The `README.md` also mentions the image name somewhere. This should be changed, as well.